packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+<script setup lang="ts">
+import { computed, ref, useSlots, onMounted, onUpdated} from 'vue'
+import { FzTableProps } from './types'
+import { FzColumn, FzColumnSlots, FzColumnProps} from '@fiscozen/simple-table'
+import { FzIconDropdown } from '@fiscozen/dropdown';
+import { FzButton } from '@fiscozen/button';
+
+const props = withDefaults(defineProps<FzTableProps>(), {
+  pageInterval: 2,
+  activePage: 0
+})
+
+const emit = defineEmits([])
+const activePage = defineModel<number>('activePage')
+
+const slots = useSlots();
+
+const defaultSlot = slots.default?.();
+const columns =
+  defaultSlot
+    ?.filter((elem) => elem.type === FzColumn)
+    .map((column) => ({
+      props: column.props as FzColumnProps,
+      children: column.children as FzColumnSlots,
+    })) ?? [];
+
+const grid = ref()
+
+const staticClasses = ref(['grid'])
+const classes = computed(() => {
+  let res: string[] = [];
+
+  const colLength = props.actions ? columns.length + 1 : columns.length
+
+  switch (colLength) {
```

Comment:
```suggestion
  res.push(`grid-cols-${colLength}`)
```

Reply:
this does not work in tailwind sadly

Reply:
I forgot this small limitation of Tailwind 
![image](https://github.com/user-attachments/assets/bf23b44b-1cb4-4489-b746-9aff67aaf4b7)